src/components/Footer/ContactDetails/ContactDetails.spec.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ContactDetails from './ContactDetails';
+
+describe('<ContactDetails />', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = shallow(<ContactDetails />);
+  });
+
+  test('should match snapshot', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
```

Comment:
When you only have 1 test case, you don't actually need **beforeEach**
for example:
```
test('should match snapshot', () => {
  const wrapper = shallow(<ContactDetails />);
  expect(wrapper).toMatchSnapshot();
});
```